### PR TITLE
Fix deployment

### PR DESCRIPTION
**Pull request by bogdanfazakas**
Fixes # .

Changes proposed in this PR:

- upgrade to node 18
- set type module

### CODE REVIEW

File: api/address.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import type { VercelRequest, VercelResponse } from '@vercel/node'
-import { getProvider } from './_utils'
```
Comment (jamiehewitt15) [inline api/address.ts:2]: Why did you change the filename?

Comment (bogdanfazakas) [reply api/address.ts:2]: whit the config changes made it was no longer visible in the output directory of vercel. 
I can try again tho since i've made more config updates after this error popped up.

**Comment by jamiehewitt15** [comment]
This endpoint is working on the preview deployment:

image

The others all seem to work fine

**Comment by bogdanfazakas** [comment]
> This endpoint is not working on the preview deployment:
> 
> image
> 
> The others all seem to work fine

fixed 😃 , all endpoints should be up

File: api/address.ts (hunk 1 above)
Comment (bogdanfazakas) [reply api/address.ts]: reverted , seem all fine now 👍

**Review by jamiehewitt15 — APPROVED** [review]
Works well now, thanks for fixing this
